### PR TITLE
Speakers page

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: data/links.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Button from '../Components/Button';
 
 export default [
   <Link uppercase key="1" href="/location/">Location</Link>,
+  <Link uppercase key="1" href="/speakers/">Speakers</Link>,
```
Comment (pfac) [inline data/links.js]: Repeated key :warning:

File: pages/index.js [hunk 2]
```diff
@@ -24,6 +25,14 @@ const Index = () => (
       <Mirror>
         <Speakers />
       </Mirror>
+
+      <CallToAction
+        title="Want to be part of this amazing lineup?"
+        text="Call for proposals is open until May 20th"
+        href="https://www.papercall.io/mirrorconf2017"
+        buttonText="SUBMIT A TALK"
```
Comment (pfac) [inline pages/index.js]: Are we still accepting submissions?

**Review by pfac — COMMENTED** [review]
:question: 

Also, why do we have both a `package-lock.json` and a `yarn.lock`?

File: pages/index.js (hunk 2 above)
Comment (naps62) [reply pages/index.js]: nope. this was a rebase issue

File: data/links.js (hunk 1 above)
Comment (naps62) [reply data/links.js]: fixed

**Comment by naps62** [comment]
not sure, but I deleted package-lock.json

**Review by pfac — APPROVED** [review]
:man_shrugging: LGTM

**Pull request merged**